src/humanPose/draw.js

```diff
@@ -1256,6 +1258,10 @@ function updateJointsAndBones(poseRenderInstance, poseObject, timestamp) {
 
     const poseHasParent = poseObject.parent && (poseObject.parent !== 'none');
     const pose = new Pose(jointPositions, jointConfidences, timestamp, {poseObjectId: poseObject.uuid, poseHasParent: poseHasParent});
+    pose.metadata.previousPose = mostRecentPoseByObjectId[poseObject.uuid];
+    if (!poseHasParent) {
```

Comment:
I would not restrict acceleration calculation to just human objects without parent. It can be available to all of them regardless whether they are auxiliary.

Reply:
Do the auxiliary poses have their own poseObjectIds? If so, how do we properly tell which pose corresponds to the previous pose of each pose? I was under the impression that the auxiliary poses share an object Id, and jumping between them is what we're trying to avoid

Reply:
Yes, they do.  An example from my console: _HUMAN_server_Wajvnee6619_pose1__Igya99zswgi is 'fused' from _HUMAN_devicekz1gkwfb_pose1__peakszmzux1 and _HUMAN_devicej1O8vomf_pose1__Tcmbg2g9kvl (both have parent property set to  _HUMAN_server_Wajvnee6619_pose1__Igya99zswg)

Reply:
In each timestamp, we reason whether the pose for the fused 'server' object should be taken from one or the other auxiliary object. But they are all fully-fledged human pose object with own pose history.